internal/routecapture/helpers.go

```diff
@@ -111,30 +105,124 @@ func mergeBodyParams(params1, params2 []*webscan.BodyParams) []*webscan.BodyPara
 	return mergedParams
 }
 
+// Helper to parse query parameters from the URL
+func parseQueryParams(reqURL *url.URL) []*webscan.QueryParams {
+	var queryParams []*webscan.QueryParams
+	for key, values := range reqURL.Query() {
+		queryParams = append(queryParams, &webscan.QueryParams{
+			Name:          key,
+			ExampleValues: values,
+		})
+	}
+	return queryParams
+}
+
+// Helper to parse body parameters
+func parseBodyParams(postData string) []*webscan.BodyParams {
```

Comment:
Can we log something or include an error if we find something that is not JSON or form encoded? XML is unfortunately still out there and I'd be interested in having some way to understand if we're seeing it

Reply:
done